cns/restserver/internalapi_test.go

```diff
@@ -490,11 +491,6 @@ func validateNetworkRequest(t *testing.T, req cns.CreateNetworkContainerRequest)
 		t.Fatalf("Failed as Primary IPAddress doesnt match, expected:%s, actual %s", req.IPConfiguration.IPSubnet.IPAddress, actualReq.IPConfiguration.IPSubnet.IPAddress)
 	}
 
```

Comment:
why was this code removed?

Reply:
This I removed because it was becoming an issue for when testing for dualstack. Essentially it is checking to make sure the IPs from the request are now in the pool but since dualstack requires multiple NCs then any test that needs two or more NCs will fail this check because you will see double the amount of IPs in the pool once the second NC gets processed